backend/src/test/scala/com/softwaremill/bootzooka/test/TestClock.scala

```diff
@@ -4,13 +4,14 @@ import java.time.Instant
 
 import cats.effect.Clock
 import cats.effect.laws.util.TestContext
+import com.softwaremill.bootzooka.util.ClockProvider
 import com.typesafe.scalalogging.StrictLogging
 import monix.eval.Task
 import monix.execution.Scheduler.Implicits.global
 
 import scala.concurrent.duration.{Duration, FiniteDuration, MILLISECONDS}
 
-class TestClock extends StrictLogging {
+class TestClock extends ClockProvider with StrictLogging {
```

Comment:
this is more than a `TestClock` - as it uses the `TextContext` underneath. Maybe we can either simplify this impl (just storing the current time as a atomic reference), or rename to e.g. `TestContextClock` to be clear what's used?

Reply:
I've decided finally to go with simpler implementation - without test context and atomic reference. It's simply reverted state of class that already exists, with proper methods implemented. Thanks!